climada/hazard/base.py

```diff
@@ -1393,3 +1442,167 @@ def _read_att_excel(self, file_name, var_names):
         self.intensity = sparse.csr_matrix(dfr.values[:, 1:num_events + 1].transpose())
         self.fraction = sparse.csr_matrix(np.ones(self.intensity.shape,
                                                   dtype=np.float))
+
+    def concatenate(self, haz_src, append=False, centroids=None):
+        """
+        Concatenate events of several hazards of same type.
+
+        This is a wrapper method for hazard.base.concatenate_hazard
+        to ensure backwards compatibility.
+
+        Parameters
+        ----------
+        haz_src: list(climada.hazard.Hazard())
+            Hazard instances of the same hazard type
+        append : bool, optional
+            If True, append the haz_src to current object. If False,
+            replace all attributes from current objects by concatenated
+            hazards from haz_src. The default is False.
+        centtroids: climada.hazard.Centroids(), optional
+            Centroids instance on which to map the concatenated hazard.
+            Default is None.
+
+        Returns
+        -------
+        None.
+
+        """
+        if append:
+            haz_src = [self] + haz_src
+        self.__dict__ = copy.deepcopy(
+            concatenate_hazard(haz_src, centroids).__dict__
+            )
+
+
+def join_centroids(haz_list):
```

Comment:
It feels strange to have a method like this that takes hazards and returns centroids.
I'd suggest centroids in and centroids out (and maybe moving it to the centroids class)?
You'd then have to update line 1571 from
```
      centroids = join_centroids(haz_list)
```
to
```
      centroids = join_centroids([haz.centroids for haz in haz_list])
```

Reply:
Good point.

Reply:
I first did not do that because there is already the method `centroids.append`. Hence, I only made this to join centroids from hazards. 

Would you still say a join  method for `centroids` is better?

Reply:
I would say so, but this version is completely fine if you don't want to put much more time into it.

Reply:
OK, I will put it there then!